### PR TITLE
Clean inspection errors

### PR DESCRIPTION
**Pull request by jonathanshaked**
Please review changes

### CODE REVIEW

File: shellfoundry_traffic/test_helpers.py [hunk 1]
```diff
@@ -6,13 +6,14 @@
 from typing import Optional, Tuple
 
 import yaml
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as Element_Tree
```
Comment (cr-yoramshamir) [inline shellfoundry_traffic/test_helpers.py]: If you want to follow PEP 8 this should be ElementTree.=
However, ET is the convention for ElementTree.
See in https://docs.python.org/3.7/library/xml.etree.elementtree.html

File: shellfoundry_traffic/test_helpers.py [hunk 2]
```diff
@@ -6,13 +6,14 @@
...
 
 from cloudshell.api.cloudshell_api import (CloudShellAPISession, ResourceAttributesUpdateRequest, AttributeNameValue,
                                            ResourceInfo, CreateReservationResponseInfo, SetConnectorRequest)
 from cloudshell.helpers.scripts.cloudshell_dev_helpers import attach_to_cloudshell_as
 from cloudshell.shell.core.session.cloudshell_session import CloudShellSessionContext
-from cloudshell.shell.core.driver_context import (ResourceCommandContext, ResourceContextDetails, AutoLoadCommandContext,
+from cloudshell.shell.core.driver_context import (ResourceCommandContext, ResourceContextDetails,
+                                                  AutoLoadCommandContext,
                                                   ConnectivityContext, InitCommandContext, AppContext)
```
Comment (cr-yoramshamir) [inline shellfoundry_traffic/test_helpers.py]: Please try to push as much into each line, the last line should be the shortest.
Something like:
from cloudshell.shell.core.driver_context import (ResourceCommandContext, ResourceContextDetails,
                                                  AutoLoadCommandContext, ConnectivityContext, InitCommandContext,
                                                  AppContext)

File: shellfoundry_traffic/test_helpers.py [hunk 3]
```diff
@@ -64,23 +65,27 @@ def create_session_from_cloudshell_config():
 # 2nd generation helers.
 #
 
+
+xml_file = '/deployment.xml'
+
+
```
Comment (cr-yoramshamir) [inline shellfoundry_traffic/test_helpers.py]: Constant names should be all upper, rename DEPLOYMENT_XML.

File: shellfoundry_traffic/test_helpers.py [hunk 4]
```diff
@@ -207,13 +212,13 @@ def debug_attach_from_deployment(reservation_id, resource_name=None, service_nam
                             service_name=service_name)
 
 
-def _conn_and_res(session: CloudShellAPISession, family: str, model: str, address: str, attributes: dict, type: str,
+def _conn_and_res(session: CloudShellAPISession, family: str, model: str, address: str, attributes: dict, a_type: str,
```
Comment (cr-yoramshamir) [inline shellfoundry_traffic/test_helpers.py]: Rename type_, see PEP 8.

**Review by cr-yoramshamir — CHANGES_REQUESTED** [review]
See comments inline.

File: shellfoundry_traffic/test_helpers.py (hunk 1 above)
Comment (jonathanshaked) [reply shellfoundry_traffic/test_helpers.py]: Changed to ET

File: shellfoundry_traffic/test_helpers.py (hunk 2 above)
Comment (jonathanshaked) [reply shellfoundry_traffic/test_helpers.py]: Fixed.

File: shellfoundry_traffic/test_helpers.py (hunk 3 above)
Comment (jonathanshaked) [reply shellfoundry_traffic/test_helpers.py]: Changed.

File: shellfoundry_traffic/test_helpers.py (hunk 4 above)
Comment (jonathanshaked) [reply shellfoundry_traffic/test_helpers.py]: Done.

**Review by cr-yoramshamir — APPROVED** [review]

**Pull request merged**